src/transformers/models/mask2former/image_processing_mask2former.py

```diff
@@ -1004,55 +1008,108 @@ def post_process_instance_segmentation(
             scores = torch.nn.functional.softmax(mask_cls, dim=-1)[:, :-1]
             labels = torch.arange(num_classes, device=device).unsqueeze(0).repeat(num_queries, 1).flatten(0, 1)
 
-            scores_per_image, topk_indices = scores.flatten(0, 1).topk(num_queries, sorted=False)
+            # keep top-k predictions
```

Comment:
Since Mask2Former Video's image processor is separated, could you undo the changes to `image_processing_mask2former.py`?